pnpm-lock.yaml

```diff
@@ -7131,7 +7146,7 @@ packages:
     resolution: {integrity: sha512-DMy+ERcEW2q8Z2Po+WNXuw3c5YaUSFjAO5GsJqfEl7UjvtIuFKO6ZrKvcItdy98dwFI2N1tg3zNIdKaQT+aNdA==}
     engines: {node: '>=8.6'}
     dependencies:
-      braces: 3.0.2
+      braces: 3.0.3
```

Comment:
This change looks incorrect. Seems like dependabot made a direct change in the lock file.

Reply:
Okay this seems also correct, but I wonder why the specifier "^" is not mentioned in the lock file

Following dependency is listed in the package.json of micromatch@4.0.5
```
 "dependencies": {
    "braces": "^3.0.2",
    "picomatch": "^2.3.1"
  },
```

Reply:
Dependabot (and renovate for that mater) do not fully understand how to write overrides, so they just make direct lock changes.
These can work sometimes, but is not always consistent.
Such as for braces, which is always referenced as ^3 (that I have seen), this is OK.
For PNPM you can use the following cli as well:
`pnpm update braces`